external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -13,20 +12,30 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageToInclude Include="xunit.abstractions"/>
+    <PackageToInclude Include="xunit.assert"/>
+    <PackageToInclude Include="xunit.extensibility.core"/>
+    <PackageToInclude Include="xunit.extensibility.execution"/>
+    <PackageToInclude Include="xunit.runner.utility"/>
+    <PackageToInclude Include="Microsoft.xunit.netcore.extensions"/>
+    <PackageToInclude Include="Microsoft.DotNet.xunit.performance.run.core"/>
+    <PackageToInclude Include="Newtonsoft.Json"/>
+    <PackageToInclude Include="xunit.console.netcore"/>
```

Comment:
I confirmed <PackageToInclude Include="xunit.console.netcore"/> is not needed so I'll exclude but the rest is needed